includes/admin/class-wc-payments-admin.php

```diff
@@ -349,6 +349,7 @@ public function register_payments_scripts() {
 			'connect'                 => [
 				'country'            => WC()->countries->get_base_country(),
 				'availableCountries' => WC_Payments_Utils::supported_countries(),
+				'states'             => WC()->countries->get_states(),
```

Comment:
Let's actualize `client/globals.d.ts` as well.

Reply:
I'd also love to align the naming: `states` -> `availableStates`

Reply:
Sorted, thanks!